presto-hive/src/test/java/com/facebook/presto/hive/TestHiveNativeWorkersQueries.java

```diff
@@ -0,0 +1,108 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.hive;
+
+import com.facebook.presto.testing.QueryRunner;
+import com.facebook.presto.tests.AbstractTestQueryFramework;
+import com.facebook.presto.tests.DistributedQueryRunner;
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableMap;
+import org.testng.annotations.Test;
+
+import java.io.IOException;
+import java.io.UncheckedIOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.Optional;
+
+import static com.google.common.base.Preconditions.checkArgument;
+import static io.airlift.tpch.TpchTable.NATION;
+
+public class TestHiveNativeWorkersQueries
```

Comment:
Maybe annotate this as `@Experimental` so people who are unawares don't try to run it locally (I presume there's no well known way to get this to work right now?)

Reply:
Would you like me to use com.facebook.presto.spi.api.Experimental or something else? Anyone can run it locally, but they would need to build native presto worker first.

Reply:
When I look at a Testng test in our codebase, I have the expectation of simply being able to run it in IntelliJ.  Because the setup here is more complicated and, as I understand it, requires some external setup which isn't captured or documented in this test, I'm just trying to think of a way to make it clear to people that you can't just do that.

Ideally, it would be really nice to develop some tools to run these locally, presuming you have Docker or some containerization software locally running on an Intel CPU.  But that sounds really complicated, so I just though that the Experimental annotation from SPI would do.  Open to other suggestions to make it clear (perhaps also detailed comments explaining how to run locally).

Reply:
I'll add @Experimental from spi. I'd be happy to add instructions, but they require checking out private GitHub repo.